### PR TITLE
OCP Details: Replace PF3 ListView with PF4 table

### PR DESCRIPTION
**Pull request by dlabrecq**
This replaces the old PF3 ListView with a new PF4 table.

Note that this is for the OCP details page only. AWS Details will be another PR.

Fixes https://github.com/project-koku/koku-ui/issues/207

Example:
screen shot 2019-02-08 at 2 02 51 am

Known issues with PF4 table: https://github.com/patternfly/patternfly-react/issues/1337

### CODE REVIEW

File: src/pages/ocpDetails/detailsTable.tsx [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+          cells: [
+            item.label,
+            this.getMonthOverMonthCost(item, index),
+            this.getTotalCost(item, index),
+          ],
+          isOpen: false,
+          item,
+        },
+        {
+          parent: index,
```
Comment (karelhala) [inline src/pages/ocpDetails/detailsTable.tsx]: This is wrongly calculated parent. What you end up with is
```JS
[
 { cells: ['parent - 1'], isOpen: false }, // 0
 { cells: ['child - 1'], parent: 0}, // 1
 { cells: ['parent - 2'], isOpen: false }, // 2
 { cells: ['child - 2'], parent: 1 } // this is wrong, it should be number 2
]
```

To calculate parent correctly you need to use `index * 2`

File: src/locales/de.json [hunk 2]
```diff
@@ -176,7 +176,6 @@
       "memory_usage": "Usage - {{value}} GB-Hours"
     },
     "change_column_title": "Month Over Month Change",
-    "charge_column_subtitle": " (Total {{total}})",
     "charge_column_title": "Charges",
```
Comment (chambridge) [inline src/locales/de.json]: This line was changed in `en.json` but not here.

File: src/locales/es.json [hunk 3]
```diff
@@ -176,7 +176,6 @@
       "memory_usage": "Usage - {{value}} GB-Hours"
     },
     "change_column_title": "Month Over Month Change",
-    "charge_column_subtitle": " (Total {{total}})",
     "charge_column_title": "Charges",
```
Comment (chambridge) [inline src/locales/es.json]: This line was changed in `en.json` but not here.

File: src/pages/ocpDetails/detailsTable.tsx (hunk 1 above)
Comment (dlabrecq) [reply src/pages/ocpDetails/detailsTable.tsx]: Ah, much better. Thank you!

File: src/locales/de.json (hunk 2 above)
Comment (dlabrecq) [reply src/locales/de.json]: Thanks. The linter didn't catch that.

File: src/locales/es.json (hunk 3 above)
Comment (dlabrecq) [reply src/locales/es.json]: Fixed. Thank you

**Pull request merged**